plugins/com.google.cloud.tools.eclipse.ui.status/src/com/google/cloud/tools/eclipse/ui/status/PollingStatusServiceImpl.java

```diff
@@ -137,8 +140,17 @@ void refreshStatus() {
               new GcpStatus(highestSeverity, Joiner.on(", ").join(affectedServices), activeIncidents); //$NON-NLS-1$
         }
       }
+    } catch (UnknownHostException | SocketException ex) {
+      logger.log(Level.WARNING, "Cannot connect to GCP: " + ex); // $NON-NLS1$
+      currentStatus =
+          new GcpStatus(
+              Severity.ERROR, Messages.getString("cannot.connect.to.gcp"), null); //$NON-NLS1$
```

Comment:
perhaps severity warning

Reply:
These [`Severity` objects](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.ui.status/src/com/google/cloud/tools/eclipse/ui/status/Incident.java#L32) pertain to GCP incident status levels, except for ERROR which indicates a problem retrieving status from GCP.